types/ckeditor__ckeditor5-paragraph/src/paragraphcommand.d.ts

```diff
@@ -17,4 +17,6 @@ export default class ParagraphCommand extends Command {
      * will be turned to paragraphs.
      */
     execute(options?: { selection?: Selection | DocumentSelection | undefined }): void;
+
+    refresh(): void;
```

Comment:
Is defining `refresh()` here needed if it exists on the base `Command` class?

Reply:
No, it's not. I've removed it now.